src/WebJobs.Script/Description/Node/NodeFunctionInvoker.cs

```diff
@@ -348,70 +351,14 @@ protected override void Dispose(bool disposing)
 
         private async Task<Dictionary<string, object>> CreateScriptExecutionContextAsync(object input, DataType dataType, TraceWriter traceWriter, FunctionInvocationContext invocationContext)
         {
-            // create a TraceWriter wrapper that can be exposed to Node.js
-            var log = (ScriptFunc)(p =>
```

Comment:
The log and metric functions I pushed out into static factory methods, to prevent regressions. The factory functions take WeakReferences, so we won't accidentally regress things going forward.